### PR TITLE
Update Git to vfs-2.21.0

### PR DESCRIPTION
**Pull request by derrickstolee**
Updates Git to 2.21.0, as rebased in microsoft/git#122.

### CODE REVIEW

File: GVFS/GVFS.Build/GVFS.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup Label="Parameters">
     <GVFSVersion>0.2.173.2</GVFSVersion>
-    <GitPackageVersion>2.20190312.1</GitPackageVersion>
+    <GitPackageVersion>2.20190513.1-pr</GitPackageVersion>
```
Comment (dscho) [inline GVFS/GVFS.Build/GVFS.props]:
```suggestion
    <GitPackageVersion>2.20190521.1-pr</GitPackageVersion>
```

(provided that [the build](https://dev.azure.com/gvfs/ci/_build/results?buildId=11485) succeeds...)

Comment (dscho) [reply GVFS/GVFS.Build/GVFS.props]: The build succeeded ;-)

Comment (derrickstolee) [reply GVFS/GVFS.Build/GVFS.props]: I applied the suggestion from my phone.

Comment (derrickstolee) [reply GVFS/GVFS.Build/GVFS.props]: I applied the suggestion from my phone.

Comment (derrickstolee) [reply GVFS/GVFS.Build/GVFS.props]: Thanks, @dscho! That worked!

**Pull request merged**